client/src/main/java/org/apache/rocketmq/client/consumer/store/RemoteBrokerOffsetStore.java

```diff
@@ -82,9 +82,8 @@ public long readOffset(final MessageQueue mq, final ReadOffsetType type) {
                     AtomicLong offset = this.offsetTable.get(mq);
                     if (offset != null) {
                         return offset.get();
-                    } else if (ReadOffsetType.READ_FROM_MEMORY == type) {
-                        return -1;
```

Comment:
offset  will be got from remote broker if it was got failed in local offset table in MEMORY_FIRST_THEN_STORE model.
Just return -1 in READ_FROM_MEMORY model.